index.html

```diff
@@ -1,33 +1,40 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+<meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
 <link rel="preconnect" href="https://fonts.googleapis.com">
 <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
 <link href="https://fonts.googleapis.com/css2?family=Inter&family=Roboto&display=swap" rel="stylesheet">
-  <link rel="preconnect" href="https://fonts.googleapis.com">
-  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-  <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>David Isea Portfolio</title>
-    <link rel="stylesheet" href="style.css">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+<title>David Isea Portfolio</title>
+<link rel="stylesheet" href="style.css">    
 </head>
 <body>
     <header class="main-header">
-        <nav class="nav-bar">
-          <a class="logo">AlucardSanin</a>
-          <img class="menu" src="Images/Unionw.svg " alt="Union" />
+      <section id="menu" class="mobile-menu">
+          <ul>
+            <li id="cls"><a href="#"><img id="close" src="Images/x.svg" alt="Close"></a></li>
+            <li id="portf"><a href="#headline">Portfolio</a></li>         
+            <li id="abo"><a href="#about">About</a></li>
```

Comment:
Your about and portfolio links from both the mobile nav menu and desktop navbar are not redirecting to the section they should be. Consider fixing it.